scripts/pi-hole/php/FTL.php

```diff
@@ -30,7 +31,7 @@ function connectFTL($address, $port) {
     if ($address == DEFAULT_FTL_IP) {
         $config = piholeFTLConfig();
         // Read port
-        $portfileName = isset($config['PORTFILE']) ? $config['PORTFILE'] : '';
+        $portfileName = isset($config['PORTFILE']) ? $config['PORTFILE'] : DEFAULT_FTL_PORTFILE;
         if ($portfileName != '') {
```

Comment:
Should we keep this check? `portfileName` should not be empty in any case now?

Reply:
What happens if a user set (by mistake or on purpose) an empty value?
```
PORTFILE=
```

Reply:
Good point.